### ISSUE TITLE
Provide advanced option to disable extra debug flags

### ISSUE THREAD

**Issue by emidln-imc** [bazelbuild/intellij#4113]
[Extra debug flags](https://github.com/bazelbuild/intellij/blob/af4ee63c17ab823a2c8bb45d1d7562a65b632665/clwb/src/com/google/idea/blaze/clwb/run/BlazeCidrRunConfigurationRunner.java#L122)  can cause rebuilds due to flag changes between CLion and bazel. It would be handy for an advanced (default off) registry setting or experiment to turn off these extra flags for users who handle flags via .bazelrc configs.

**Comment by keith** [comment]
seems like this was fixed by https://github.com/bazelbuild/intellij/pull/4913 with `bazel.clwb.debug.extraflags.disabled` (i don't know where this setting goes tho)

**Comment by tpasternak** [comment]
@keith yeah, either in settings or in the projectview file (even better because it's project-specifc). Would you be interested in contributing? Personally I don't have capacity to work on it
